Clean up flow triggers MultipleCondition

### Describe the issue

- [ ] MultipleCondition example, description and properties need to be fixed
- [ ] the return value of "io.kestra.core.models.triggers.TimeWindow.getType()" is null
	at io.kestra.core.validations.validator.TimeWindowValidator.isValid(TimeWindowValidator.java:26)
	at io.kestra.core.validations.validator.TimeWindowValidator.isValid(TimeWindowValidator.java:13)

`jakarta.validation.ValidationException: HV000028: Unexpected exception during isValid call.
	at org.hibernate.validator.internal.engine.constraintvalidation.ConstraintTree.validateSingleConstraint(ConstraintTree.java:186)
	at org.hibernate.validator.internal.engine.constraintvalidation.SimpleConstraintTree.validateConstraints(SimpleConstraintTree.java:66)
	at org.hibernate.validator.internal.engine.constraintvalidation.ConstraintTree.validateConstraints(ConstraintTree.java:75)
	at org.hibernate.validator.internal.metadata.core.MetaConstraint.doValidateConstraint(MetaConstraint.java:130)
	at org.hibernate.validator.internal.metadata.core.MetaConstraint.validateConstraint(MetaConstraint.java:123)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateMetaConstraint(ValidatorImpl.java:555)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateConstraintsForSingleDefaultGroupElement(ValidatorImpl.java:518)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateConstraintsForDefaultGroup(ValidatorImpl.java:488)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateConstraintsForCurrentGroup(ValidatorImpl.java:450)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateInContext(ValidatorImpl.java:400)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateCascadedAnnotatedObjectForCurrentGroup(ValidatorImpl.java:629)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateCascadedConstraints(ValidatorImpl.java:590)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateInContext(ValidatorImpl.java:409)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateCascadedAnnotatedObjectForCurrentGroup(ValidatorImpl.java:629)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateCascadedConstraints(ValidatorImpl.java:590)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateInContext(ValidatorImpl.java:409)
	at org.hibernate.validator.internal.engine.ValidatorImpl$CascadingValueReceiver.doValidate(ValidatorImpl.java:715)
	at org.hibernate.validator.internal.engine.ValidatorImpl$CascadingValueReceiver.indexedValue(ValidatorImpl.java:681)
	at org.hibernate.validator.internal.engine.valueextraction.ListValueExtractor.extractValues(ListValueExtractor.java:28)
	at org.hibernate.validator.internal.engine.valueextraction.ListValueExtractor.extractValues(ListValueExtractor.java:17)
	at org.hibernate.validator.internal.engine.valueextraction.ValueExtractorHelper.extractValues(ValueExtractorHelper.java:42)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateCascadedContainerElementsForCurrentGroup(ValidatorImpl.java:651)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateCascadedConstraints(ValidatorImpl.java:598)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateInContext(ValidatorImpl.java:409)
	at org.hibernate.validator.internal.engine.ValidatorImpl.validate(ValidatorImpl.java:172)
	at io.micronaut.configuration.hibernate.validator.MicronautHibernateValidator.validate(MicronautHibernateValidator.java:69)
	at io.kestra.core.models.validations.ModelValidator.isValid(ModelValidator.java:25)
	at io.kestra.core.models.validations.ModelValidator.validate(ModelValidator.java:18)
	at io.kestra.webserver.controllers.api.FlowController.lambda$validateFlows$11(FlowController.java:606)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source)
	at io.kestra.webserver.controllers.api.FlowController.validateFlows(FlowController.java:620)
	at io.kestra.webserver.controllers.api.$FlowController$Definition$Intercepted.$$access$$validateFlows(Unknown Source)
	at io.kestra.webserver.controllers.api.$FlowController$Definition$Exec.dispatch(Unknown Source)
	at io.micronaut.context.AbstractExecutableMethodsDefinition$DispatchedExecutableMethod.invoke(AbstractExecutableMethodsDefinition.java:456)
	at io.micronaut.aop.chain.MethodInterceptorChain.proceed(MethodInterceptorChain.java:134)
	at io.micronaut.validation.ValidatingInterceptor.validateReturnExecutableValidator(ValidatingInterceptor.java:166)
	at io.micronaut.validation.ValidatingInterceptor.intercept(ValidatingInterceptor.java:109)
	at io.micronaut.aop.chain.MethodInterceptorChain.proceed(MethodInterceptorChain.java:143)
	at io.kestra.webserver.controllers.api.$FlowController$Definition$Intercepted.validateFlows(Unknown Source)
	at io.kestra.webserver.controllers.api.$FlowController$Definition$Exec.dispatch(Unknown Source)
	at io.micronaut.context.AbstractExecutableMethodsDefinition$DispatchedExecutableMethod.invokeUnsafe(AbstractExecutableMethodsDefinition.java:461)
	at io.micronaut.context.DefaultBeanContext$BeanContextUnsafeExecutionHandle.invokeUnsafe(DefaultBeanContext.java:4354)
	at io.micronaut.web.router.AbstractRouteMatch.execute(AbstractRouteMatch.java:272)
	at io.micronaut.web.router.DefaultUriRouteMatch.execute(DefaultUriRouteMatch.java:38)
	at io.micronaut.http.server.RouteExecutor.executeRouteAndConvertBody(RouteExecutor.java:488)
	at io.micronaut.http.server.RouteExecutor.lambda$callRoute$5(RouteExecutor.java:465)
	at io.micronaut.core.execution.ExecutionFlow.lambda$async$1(ExecutionFlow.java:87)
	at io.micrometer.core.instrument.composite.CompositeTimer.record(CompositeTimer.java:141)
	at io.micrometer.core.instrument.Timer.lambda$wrap$2(Timer.java:198)
	at io.micronaut.core.propagation.PropagatedContext.lambda$wrap$3(PropagatedContext.java:211)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NullPointerException: Cannot invoke "io.kestra.core.models.triggers.TimeWindow$Type.ordinal()" because the return value of "io.kestra.core.models.triggers.TimeWindow.getType()" is null
	at io.kestra.core.validations.validator.TimeWindowValidator.isValid(TimeWindowValidator.java:26)
	at io.kestra.core.validations.validator.TimeWindowValidator.isValid(TimeWindowValidator.java:13)
	at io.micronaut.validation.validator.constraints.ConstraintValidator.isValid(ConstraintValidator.java:65)
	at org.hibernate.validator.internal.engine.constraintvalidation.ConstraintTree.validateSingleConstraint(ConstraintTree.java:180)`


Repro:
```yaml
id: new
namespace: company.team

tasks:
  - id: hello
    type: io.kestra.plugin.core.log.Log
    message: Hello World! 

triggers:
  - id: multiple-listen-flow
    type: io.kestra.plugin.core.trigger.Flow
    preconditions:
      id: flowAandB
      timeWindow:
        window: PT12H
      flows:
        - flowId: flowA
          namespace: dev
        - flowId: flowB
          namespace: dev          
```

### Environment

- Kestra Version: develop

--- a/core/src/main/java/io/kestra/core/models/triggers/TimeWindow.java
+++ b/core/src/main/java/io/kestra/core/models/triggers/TimeWindow.java
@@ -3,7 +3,6 @@ package io.kestra.core.models.triggers;
 import io.kestra.core.models.annotations.PluginProperty;
 import io.kestra.core.validations.TimeWindowValidation;
 import io.swagger.v3.oas.annotations.media.Schema;
-import jakarta.validation.constraints.NotNull;
 import lombok.Builder;
 import lombok.Getter;
 import lombok.With;
@@ -19,7 +18,6 @@ public class TimeWindow {
         title = "The type of the SLA",
         description = "The default SLA is a sliding window (`DURATION_WINDOW`) with a window of 24 hours."
     )
-    @NotNull
     @Builder.Default
     @PluginProperty
     private TimeWindow.Type type = TimeWindow.Type.DURATION_WINDOW;

--- a/core/src/main/java/io/kestra/core/models/triggers/multipleflows/MultipleConditionStorageInterface.java
+++ b/core/src/main/java/io/kestra/core/models/triggers/multipleflows/MultipleConditionStorageInterface.java
@@ -11,6 +11,8 @@ import java.util.HashMap;
 import java.util.List;
 import java.util.Optional;
 
+import static io.kestra.core.models.triggers.TimeWindow.Type.DURATION_WINDOW;
+
 public interface MultipleConditionStorageInterface {
     Optional<MultipleConditionWindow> get(Flow flow, String conditionId);
 
@@ -20,7 +22,8 @@ public interface MultipleConditionStorageInterface {
         ZonedDateTime now = ZonedDateTime.now().withNano(0);
         TimeWindow timeWindow = multipleCondition.getTimeWindow() != null ? multipleCondition.getTimeWindow() : TimeWindow.builder().build();
 
-        var startAndEnd = switch (timeWindow.getType()) {
+        TimeWindow.Type type = timeWindow.getType() != null ? timeWindow.getType() : DURATION_WINDOW;
+        var startAndEnd = switch (type) {
             case DURATION_WINDOW -> {
                 Duration window = timeWindow.getWindow() == null ? Duration.ofDays(1) : timeWindow.getWindow();
                 if (window.toDays() > 0) {

--- a/core/src/main/java/io/kestra/core/validations/validator/TimeWindowValidator.java
+++ b/core/src/main/java/io/kestra/core/validations/validator/TimeWindowValidator.java
@@ -10,6 +10,8 @@ import io.micronaut.validation.validator.constraints.ConstraintValidator;
 import io.micronaut.validation.validator.constraints.ConstraintValidatorContext;
 import jakarta.inject.Singleton;
 
+import static io.kestra.core.models.triggers.TimeWindow.Type.DURATION_WINDOW;
+
 @Singleton
 @Introspected
 public class TimeWindowValidator implements ConstraintValidator<TimeWindowValidation, TimeWindow> {
@@ -23,7 +25,8 @@ public class TimeWindowValidator implements ConstraintValidator<TimeWindowValida
             return true;
         }
 
-        return switch (value.getType()) {
+        TimeWindow.Type type = value.getType() != null ? value.getType() : DURATION_WINDOW;
+        return switch (type) {
             case DAILY_TIME_DEADLINE -> {
                 if (value.getWindow() != null || value.getWindowAdvance() != null  || value.getStartTime() != null || value.getEndTime() != null) {
                     context.disableDefaultConstraintViolation();

--- a/core/src/main/java/io/kestra/plugin/core/condition/MultipleCondition.java
+++ b/core/src/main/java/io/kestra/plugin/core/condition/MultipleCondition.java
@@ -28,9 +28,9 @@ import java.util.*;
 @Schema(
     title = "Run a flow if the list of preconditions are met in a time window.",
     description = """
-        **This task is deprecated**, use io.kestra.plugin.core.condition.ExecutionsWindow or io.kestra.plugin.core.condition.FilteredExecutionsWindow instead.
+        **This task is deprecated**, use the `preconditions` property of the `io.kestra.plugin.core.trigger.Flow` trigger instead.
         Will trigger an executions when all the flows defined by the preconditions are successfully executed in a specific period of time.
-        The period is defined by the `timeSLA` property and is by default a duration window of 24 hours."""
+        The period is defined by the `timeWindow` property and is by default a duration window of 24 hours."""
 )
 @Plugin(
     examples = {
@@ -47,7 +47,7 @@ import java.util.*;
                 "        - SUCCESS",
                 "      - id: multiple",
                 "        type: io.kestra.plugin.core.condition.MultipleCondition",
-                "        sla:",
+                "        timeWindow:",
                 "          window: PT12H",
                 "        conditions:",
                 "          flow-a:",
@@ -102,7 +102,7 @@ public class MultipleCondition extends Condition implements io.kestra.core.model
 
     @Schema(
         title = "The duration of the window",
-        description = "Deprecated, use `timeSLA.window` instead.")
+        description = "Deprecated, use `timeWindow.window` instead.")
     @PluginProperty
     @Deprecated
     private Duration window;
@@ -114,7 +114,7 @@ public class MultipleCondition extends Condition implements io.kestra.core.model
 
     @Schema(
         title = "The window advance duration",
-        description = "Deprecated, use `timeSLA.windowAdvance` instead.")
+        description = "Deprecated, use `timeWindow.windowAdvance` instead.")
     @PluginProperty
     @Deprecated
     private Duration windowAdvance;

